aea/cli_gui/__init__.py

```diff
@@ -446,12 +446,12 @@ def favicon():
     return app
 
 
-def run(port: int):
```

Comment:
@8ball030 please set a default arg on host `127.0.0.1` for backwards compatibility

Reply:
This should do it! :)